l1-contracts/test/Rollup.t.sol

```diff
@@ -726,9 +725,55 @@ contract RollupTest is DecoderBase, TimeFns {
     assertEq(rollup.getProvenBlockNumber(), 0 + toProve, "Invalid proven block number");
   }
 
+  function testRevertBlocksAcrossEpochs() public setUpFor("mixed_block_1") {
```

Comment:
Lets update the name of this test - it is not clear that this is submitting proofs for blocks across epochs rather than proposing two blocks across epochs

testRevertSubmittingProofForBlocksAcrossEpochs